app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    words = command.split()
+    file1 = words[1]
+    file2 = words[2]
```

Comment:
check if the command is passed correctly

Reply:
do it

Reply:
Sorry, I don't understand what you mean.